.gitignore

```diff
@@ -5,6 +5,7 @@
 *.iml
 /traefik
 /traefik.toml
+vendor/
```

Comment:
What did point 2 refer to again? Seems like the comment was placed at the wrong spot, or the Github UI is fooling us once again.